db/migrate/20210109124000_create_planets.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePlanets < ActiveRecord::Migration[6.0]
+  def change
+    create_table :planets do |t|
+      t.string :name
+      t.string :description
```

Comment:
Maybe change the description as text and not string.

Reply:
Yup, shouldn't be a string